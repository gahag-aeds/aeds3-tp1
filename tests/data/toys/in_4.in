5
0.0000 0.54148 0.56548 0.35158 0.5086 
0.45852 0.0000 0.33925 0.43014 0.55654 
0.43452 0.66075 0.0000 0.93287 0.62086 
0.64842 0.56986 0.06713 0.0000 0.01161 
0.4914 0.44346 0.37914 0.98839 0.0000 
