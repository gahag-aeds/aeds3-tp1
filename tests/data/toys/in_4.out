0.206663 0.154530 0.345379 0.064230 0.229199 